src/pages/Vaults/Vault/components/CollateralForm/CollateralForm.tsx

```diff
@@ -220,9 +213,9 @@ const CollateralForm = ({
       return 'Please enter an amount greater than 1 Planck';
     }
 
-    if (collateralBalance && collateralTokenAmount.gt(collateralBalance.transferable)) {
-      return t(`Must be less than ${collateralToken.ticker} balance!`);
-    }
+    // if (collateralBalance && collateralTokenAmount.gt(collateralBalance.transferable)) {
+    //   return t(`Must be less than ${collateralToken.ticker} balance!`);
+    // }
```

Comment:
Why is this code commented out?

Reply:
This Form is not implemented yet, but might be in the future, when we implement the new modals for vault dashboard.

Reply:
Ah okay, thanks for clarifying, can you add comment there about this for the future reference please? :)